### PR TITLE
New ACA Features - Workload Profiles

### PR DESCRIPTION
**Pull request by thotheod**
Firewall deployment in the hub
add diagnostics for CAE
Add custom name for Managed Resources RG
fix DNS Zones
[New Container Apps Workload Profiles](https://techcommunity.microsoft.com/t5/apps-on-azure-blog/generally-available-azure-container-apps-workload-profiles-more/ba-p/3913345)

### CODE REVIEW

File: scenarios/shared/bicep/aca-environment.bicep [hunk 1]
```diff
@@ -15,46 +15,91 @@ param tags object = {}
 param zoneRedundant bool = false
 
 @description('Mandatory, default is Consumption')
-@allowed([
-  'Consumption'
-  'Premium'
-])
-param sku string= 'Consumption'
+param workloadProfiles array = [
```
Comment (aarthiem) [inline scenarios/shared/bicep/aca-environment.bicep:18]: can we add guidance or comment to show how to add dedicated

File: docs/design-areas/security.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Security is a one of the foundational design principles and also a key design ar
...
+- Azure Container Apps allows for _internal_ and _external_ connectivity models. This allows for connecting the Container Apps to the outside world directly or not at all. In this Landing Zone Accelerator, we recommend the _internal_ model and assume that the application will be either fully internal facing or will be published externally through a service that adds the appropriate security controls, such as Azure Application Gateway or Azure Front Door. If you want to secure outbound traffic, you can configure a route table to instruct all the outgoing traffic to go through an Azure Firewall that is deployed in the Hub network. The security architecture would then look as follows (the boxes with Application Gateway and Azure Front Door being optional in internal-facing scenarios):
```
Comment (aarthiem) [inline docs/design-areas/security.md:12]: can we update the networking doc as well

**Review by aarthiem — COMMENTED** [review]
Thanks Thodoris, Minor comments added.

File: scenarios/shared/bicep/aca-environment.bicep (hunk 1 above)
Comment (thotheod) [reply scenarios/shared/bicep/aca-environment.bicep:18]: sure, I will re-work that part

Comment (thotheod) [reply scenarios/shared/bicep/aca-environment.bicep:18]: I ll do it

File: docs/design-areas/security.md (hunk 2 above)
Comment (thotheod) [reply docs/design-areas/security.md:12]: sure

**Review by ibersanoMS — APPROVED** [review]
Looks good - thanks!

File: scenarios/aca-internal/bicep/modules/01-hub/modules/azureFirewall.bicep [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+        type: 'allow'
+      }
+      priority: 130
+      rules: [ 
+        {          
+          name: 'allow-developer-services'
+          fqdnTags: [ ]
+          targetFqdns: [
+            'github.com'
+            '*.github.com'
```
[the remark this answers is not in the document]
Comment (pankajagrawal16) [reply scenarios/aca-internal/bicep/modules/01-hub/modules/azureFirewall.bicep:168]: Also githubusercontent.com,*.githubusercontent.com

**Review by aarthiem — APPROVED** [review]

**Pull request merged**